app/forms/curation_concerns/generic_work_form.rb

```diff
@@ -8,7 +8,7 @@ class GenericWorkForm < Umrdr::Forms::WorkForm
     self.required_fields = [ :title, :creator, :methodology, :description, :rights, :subject ]
 
     def rendered_terms
-      [ :title, :creator, :methodology, :description, :date_coverage, :rights, :subject, :keyword, :language, :resource_type, :isReferencedBy ]
+      [ :title, :creator, :methodology, :description, :date_coverage, :rights, :subject, :keyword, :language, :resource_type, :isReferencedBy, :visibility ]
```

Comment:
I would use `is_draft?` or 'draft_status' as the form parameter for the checkbox.  It more closely matches the language of the text box, and it avoids the case where visibility could be assigned the value of 'true'.